src/zmq/zmqpublishnotifier.cpp

```diff
@@ -6,21 +6,37 @@
 
 #include <chain.h>
 #include <chainparams.h>
+#include <crypto/common.h>
+#include <logging.h>
+#include <netaddress.h>
 #include <netbase.h>
 #include <node/blockstorage.h>
+#include <primitives/block.h>
+#include <primitives/transaction.h>
 #include <rpc/server.h>
+#include <serialize.h>
 #include <streams.h>
-#include <util/system.h>
+#include <sync.h>
+#include <uint256.h>
+#include <version.h>
 #include <zmq/zmqutil.h>
 
 #include <zmq.h>
 
+#include <cassert>
 #include <cstdarg>
 #include <cstddef>
+#include <cstdint>
 #include <map>
 #include <optional>
+#include <string.h>
```

Comment:
Given you're introducing `c*` headers everywhere else, you can use `cstring` here.

Reply:
Everywhere in our codebase we use `memcpy` and `strlen` which are declared in the global namespace rather `std::` one. Therefore, the C-header `string.h` is used.

Replacements s/`memcpy`/`std::memcpy`/, s/`strlen`/`std::strlen`/ and s/`string.h`/`ctring`/ deserve its own PR, aren't they?

Reply:
> Everywhere in our codebase we use memcpy and strlen which are declared in the global namespace rather std:: one.

We use both `memcpy` and `std::memcpy` in the codebase. Given we can use `memcpy` & `std::memcpy` with `cstring`, but only `memcpy` with `string.h`, and given that we are migrating towards the cpp headers, I don't see why we'd prefer `string.h`, or newly add it now, just to change it later, when we can just use `cstring` and acheive the same thing.

Reply:
Thanks. [Updated](https://github.com/bitcoin/bitcoin/pull/26254#issuecomment-1273430487).